setup.sql

```diff
@@ -0,0 +1,147 @@
+CREATE DATABASE greatauthors;
+
+CREATE USER 'greatauthors'@'localhost' IDENTIFIED BY 'b3tt3rth@nth3m0v13!';
+
+GRANT ALL ON greatauthors.* TO 'greatauthors'@'localhost';
+
+USE greatauthors;
+
+CREATE TABLE authors (
+id INT auto_increment,
+nameFirst VARCHAR(255),
+nameLast VARCHAR(255),
+createdAt DATETIME DEFAULT NOW(),
+updatedAt DATETIME DEFAULT NOW() ON UPDATE NOW(),
+deletedAt DATETIME,
+PRIMARY KEY(id)
+);
+
+CREATE TABLE genres (
+id INT auto_increment,
+name VARCHAR(255),
+createdAt DATETIME DEFAULT NOW(),
+updatedAt DATETIME DEFAULT NOW() ON UPDATE NOW(),
+deletedAt DATETIME,
+PRIMARY KEY(id)
+);
+
+CREATE TABLE novels (
+id INT auto_increment,
+title VARCHAR(255),
+createdAt DATETIME DEFAULT NOW(),
+updatedAt DATETIME DEFAULT NOW() ON UPDATE NOW(),
+deletedAt DATETIME,
+PRIMARY KEY(id),
+FOREIGN KEY(authorId) REFERENCES authors(id),
```

Comment:
Extra comma here throws an error. Also you never made the authorId column in this table.